### PR TITLE
Clean up post nhrc

### PR DESCRIPTION
**Pull request by ericcanton**
* Removes the module `pisces.models` whose models proved more cumbersome than helpful.
* Updates and cleans up analysis scripts.
* Rewrites README to clarify our intents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,23 @@
 # pisces
 
-
 <!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
```
Comment (ojwalch) [inline README.md:3]: ^^ Just cause I'm curious, is it still the case that the file is autogenerated?

File: README.md [hunk 2]
```diff
@@ -51,32 +34,34 @@ pip install -e .
 ### Common issues
 
 You may end up with a version of Keras incompatible with the marshalled
-data in `pisces/cached_models`. In that case, re-run the generation
-notebook `<pisces>/analyses/convert_mads_olsen_model_to_keras.ipynb`
+data in `pisces/cached_models`. In that case, run
+[`pisces_setup`](https://Arcascope.github.io/pisces/mads_olsen_support.html#pisces_setup)
```
Comment (ojwalch) [inline README.md:38]: Propose renaming `mads_olsen_support.html` to `deep_unet_support.html`

File: README.md [hunk 3]
```diff
@@ -51,32 +34,34 @@ pip install -e .
...
+is in your path as long as a Python environment with `pisces` installed
+is active.
 
 ## Usage
 
-The primary module to import is `pisces.experiments` which contains
-classes used for discovering and providing access to data sets in your
-chosen folder, as well as (trainable) classifiers.
+The pipeline is intended to be flexible and can be easily extended to
+include new models, datasets, and evaluation metrics. In version 2.0, we
```
Comment (ojwalch) [inline README.md:47]: Propose changing to `In version 2.0, we have streamlined the library to prioritizing nimbleness and easy debugging.`

File: examples/NHRC/nhrc_utils/analysis.py [hunk 4]
```diff
@@ -11,8 +11,8 @@
 import numpy as np
 import tensorflow as tf
 
-from analyses.NHRC.nhrc_utils.model_definitions import LR_POST_PAD, LR_PRE_PAD, EXTRA_LOWER, LR_LOWER, cnn_pred_proba, lr_cnn_pred_proba, naive_pred_proba
-from pisces.models import load_saved_keras
+from examples.NHRC.nhrc_utils.model_definitions import LR_POST_PAD, LR_PRE_PAD, EXTRA_LOWER, LR_LOWER, cnn_pred_proba, lr_cnn_pred_proba, naive_pred_proba
+from pisces.mads_olsen_support import load_saved_keras
```
Comment (ojwalch) [inline examples/NHRC/nhrc_utils/analysis.py:15]: Here again propose renaming to `deep_unet_support`

**Review by ojwalch — COMMENTED** [review]
Few suggested renamings/tweaks to README; otherwise, looks great!

File: README.md (hunk 1 above)
Comment (ericcanton) [reply README.md:3]: It is! generated using `nbdev_docs` from `nbs/index.ipynb`

**Review by ojwalch — APPROVED** [review]
merge this dude

**Pull request merged**